easybuild/easyconfigs/g/GCCcore/GCCcore-6.3.0.eb

```diff
@@ -37,6 +37,7 @@ builddependencies = [
 patches = [
     ('mpfr-%s-allpatches-20161215.patch' % mpfr_version, '../mpfr-%s' % mpfr_version),
     'GCCcore-6.2.0-fix-find-isl.patch',
+    'GCCcore-6.3.0-fix-ubsan.patch',
```

Comment:
@zao Please use `%(version)s` when the version in the patch name matches the `version`, rather than hardcoding it.

I assume this patch is irrelevant for GCC 6.4.0 (#4794)?